notebooks/Liquidators_data.ipynb

```diff
@@ -77,100 +80,82 @@
     }
    ],
    "source": [
-    "def load_data_from_google_storage(url):\n",
-    "    \"\"\"Load data from a Google Storage URL using pandas.read_parquet.\"\"\"\n",
-    "    response = requests.get(url)\n",
-    "    response.raise_for_status()  # Ensure the request was successful\n",
-    "\n",
-    "    with BytesIO(response.content) as f:\n",
-    "        df = pd.read_parquet(f)\n",
-    "    return df\n",
-    "\n",
-    "def load_data_from_local_parquet(file_path):\n",
-    "    \"\"\"Load data from a local Parquet file using pandas.read_parquet.\"\"\"\n",
-    "    df = pd.read_parquet(file_path)\n",
-    "    return df\n",
-    "\n",
-    "def load_protocol_data(source, config):\n",
-    "    \"\"\"Load data based on the specified source.\"\"\"\n",
+    "\n",
+    "def load_protocol_data(protocols: List[str]) -> pd.DataFrame:\n",
```

Comment:
```suggestion
    "def load_protocol_data(protocols: list[str]) -> pd.DataFrame:\n",
```
With the new Python, we can do type annotations like this. Holds also for dict, tuple, etc.